[2025-02-12T23:35:31.660+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T23:35:31.686+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:32:00+00:00 [queued]>
[2025-02-12T23:35:31.700+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:32:00+00:00 [queued]>
[2025-02-12T23:35:31.701+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T23:35:31.720+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): teste> on 2025-02-12 23:32:00+00:00
[2025-02-12T23:35:31.731+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_omdb_api', 'teste', 'scheduled__2025-02-12T23:32:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/call_api.py', '--cfg-path', '/tmp/tmp5u8996ob']
[2025-02-12T23:35:31.733+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask teste
[2025-02-12T23:35:31.730+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=70) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-12T23:35:31.736+0000] {standard_task_runner.py:72} INFO - Started process 76 to run task
[2025-02-12T23:35:31.830+0000] {task_command.py:467} INFO - Running <TaskInstance: call_omdb_api.teste scheduled__2025-02-12T23:32:00+00:00 [running]> on host 76fc25d1410f
[2025-02-12T23:35:32.034+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Celso Cazetto' AIRFLOW_CTX_DAG_ID='call_omdb_api' AIRFLOW_CTX_TASK_ID='teste' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T23:32:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-12T23:32:00+00:00'
[2025-02-12T23:35:32.037+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T23:35:32.103+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-12T23:35:32.105+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'source /home/***/venv/omdb_env/bin/activate && python /opt/***/pipeline/main.py']
[2025-02-12T23:35:32.124+0000] {subprocess.py:99} INFO - Output:
[2025-02-12T23:35:34.224+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,224 - INFO - ESTOU RODANDO ESSA PORRA
[2025-02-12T23:35:34.225+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,225 - INFO - Carregando o arquivo de configuração
[2025-02-12T23:35:34.226+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,226 - INFO - Buscando os dados de filme do ID:tt0499549
[2025-02-12T23:35:34.549+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,549 - INFO - Dados da API carregados com sucesso
[2025-02-12T23:35:34.550+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,549 - INFO - Iniciando transformação dos dados.
[2025-02-12T23:35:34.559+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,559 - INFO - Estabelecendo conexão com o banco de dados:
[2025-02-12T23:35:34.730+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,730 - INFO - Conexão estabelecida com sucesso.
[2025-02-12T23:35:34.818+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,818 - INFO - Iniciando o inserção na tabela: omdb.filmes
[2025-02-12T23:35:34.833+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,833 - INFO - 0 Inseridos na tabela omdb.filmes
[2025-02-12T23:35:34.839+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,839 - INFO - Iniciando o inserção na tabela: omdb.genero
[2025-02-12T23:35:34.846+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,845 - INFO - 0 Inseridos na tabela omdb.genero
[2025-02-12T23:35:34.863+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,863 - INFO - Iniciando o inserção na tabela: omdb.filme_genero
[2025-02-12T23:35:34.868+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,867 - INFO - 0 Inseridos na tabela omdb.filme_genero
[2025-02-12T23:35:34.874+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,874 - INFO - Conexão finalizada com o banco
[2025-02-12T23:35:34.875+0000] {subprocess.py:106} INFO - 2025-02-12 23:35:34,875 - INFO - Fim do processo de ingestão.
[2025-02-12T23:35:34.876+0000] {subprocess.py:106} INFO - 2.0.38
[2025-02-12T23:35:35.387+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-12T23:35:35.431+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T23:35:35.437+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=call_omdb_api, task_id=teste, run_id=scheduled__2025-02-12T23:32:00+00:00, execution_date=20250212T233200, start_date=20250212T233531, end_date=20250212T233535
[2025-02-12T23:35:35.531+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T23:35:35.551+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T23:35:35.553+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
